Backend/src/main/java/ca/mcgill/cooperator/controller/CourseOfferingController.java

```diff
@@ -61,6 +66,22 @@ public CourseOfferingDto createCourseOfferingDto(
                         courseOfferingDto.getYear(), courseOfferingDto.getSeason(), course);
         return ControllerUtils.convertToDto(courseOffering);
     }
+    
+    @GetMapping("/seasons")
+    public Season[] getAllCoopSeasons() {
+        Season[] seasons = Season.values();
+        return seasons;
+    }
+    
+    @GetMapping("/years")
+    public int[] getYearsToDisplay() {
```

Comment:
Making a backend call for this is probably overkill, we can just make a function for this in the frontend (in `boot/common.js`)